packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -1614,15 +1615,21 @@ export class Bucket extends BucketBase {
     }
     Bucket.validateBucketName(bucketName);
 
+    const _regionInfo = regionInfo.RegionInfo.get(region);
```

Comment:
```suggestion
    const regionInfo = regionInfo.RegionInfo.get(region);
```

Reply:
Oh I see why you did that. Basically let's avoid the `_` prefixes. You can probably change the import to not collide.